src/amoc_dist.erl

```diff
@@ -5,26 +5,39 @@
 -module(amoc_dist).
 
 -export([start_nodes/0,
+         ping_nodes/0,
          do/3,
          do/4,
          add/1,
          add/2,
          remove/2,
-         remove/3]).
+         remove/3,
+         test_status/1]).
 %% ------------------------------------------------------------------
 %% API
 %% ------------------------------------------------------------------
 -spec start_nodes() -> [ok].
 start_nodes() ->
-    Hosts = application:get_env(amoc, hosts, []),
-    Path = application:get_env(amoc, path, "/usr"),
+    Hosts = amoc_config:get(hosts, []),
+    Path = amoc_config:get(path, "/usr"),
     start_nodes(Hosts, Path).
 
+-spec ping_nodes() -> [{atom(), pong|pang}].
+ping_nodes() ->
+    Hosts = amoc_config:get(hosts, []),
+    [ {erlang:list_to_atom(Host), ping_node(amoc_slave:node_name(Host))} || Host <- Hosts ].
```

Comment:
Line 28 is too long:     [ {erlang:list_to_atom(Host), ping_node(amoc_slave:node_name(Host))} || Host <- Hosts ]..